ch2-git-basics/cloning.qmd

```diff
@@ -0,0 +1,57 @@
+## Cloning an Existing Repository
+
+### Introduction
+
+This section will guide you through the process of cloning a GitHub repository. Cloning a repository means making a local copy of a project that exists on GitHub. This enables you to make changes and test locally before pushing to be approved and merged with the main branch on GitHub.
```

Comment:
"that exists on GitHub" -> "that exists on a "remote" DevOps server such as Axure DevOps, GitHub or GitLab"